Servers/UI/OJS.Servers.Ui/ClientApp/src/components/submissions/details/SubmissionDetails.tsx

```diff
@@ -23,12 +25,43 @@ const SubmissionDetails = () => {
     } = useSubmissionsDetails();
     const { actions: { setPageTitle } } = usePageTitles();
     const { getAdministrationRetestSubmissionInternalUrl } = useAppUrls();
+    const {
+        state: { contest },
+        actions: { getContestByProblemId },
+    } = useContests();
+
+    const { getRegisterContestTypeUrl } = useAppUrls();
+
+    useEffect(() => {
+        const reload = sessionStorage.getItem('reload');
+        if (isNil(reload)) {
+            sessionStorage.reload = true;
+            window.location.reload();
+        } else {
+            sessionStorage.removeItem('reload');
+        }
+    }, []);
```

Comment:
1. What is the purpose of this?
2. Don't use `sessionStorage` directly. Use `use-services`. There is a `CacheService` implementation that takes a type of storage. Currently, only `localStorageService` is provided, but a `sessionStorageService` can be added.

Reply:
The purpose was to refresh the page once after loading, so that there is no bug with undefined values in the url. Since it was just a temporary solution, I removed it.